src/ELF/constants.jl

```diff
@@ -1,3 +1,5 @@
+# These constants are in glibc: https://github.com/bminor/glibc/blob/969d7cf89966d55fbf0732ae7bc0949a970c837d/elf/elf.h#L357
```

Comment:
It's misleading to point to a single line here (I did that in https://github.com/JuliaIO/ObjectFile.jl/pull/47#issuecomment-2558351778 because I was pointing to where the RISC-V macro is defined, relevant to that PR), also in the source code of LLVM there are better comments to explain the source of the various macros:
```suggestion
# These constants are in glibc and llvm:
# * https://github.com/bminor/glibc/blob/969d7cf89966d55fbf0732ae7bc0949a970c837d/elf/elf.h
# * https://github.com/llvm/llvm-project/blob/b2d7312d52f299a2f854c673796ad1a699e0d79f/llvm/include/llvm/BinaryFormat/ELF.h
```